DigitalLearningSolutions.Data/Helpers/LearningHubSsoSecurityHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+    using DigitalLearningSolutions.Data.Services;
+
+    public class LearningHubSsoSecurityHelper
+    {
+        private static readonly DateTime UnixEpoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        private readonly IClockService clockService;
+
+        public LearningHubSsoSecurityHelper(IClockService clockService)
+        {
+            this.clockService = clockService;
+        }
+
+        public string GenerateHash(string state, string secretKey)

```

Comment:
The secret value will eventually come from config. It probably makes sense to keep this as a parameter and to be more "helper-like" assuming the service using this helper will itself utilise config to obtain the value.

Reply:
Now that config is being injected directly into this helper service, the secret key might as well be taken from there, rather than a parameter, to be consistent.